### DISCUSSION TITLE
Github copilot

### DISCUSSION THREAD

**Discussion by Eddievin** [community/community#4415] [Copilot Conversations] [answered]
I am so much interested in starting to use github copilot but I keep getting the following message in the screenshot, is there a version that will work okay for me without hustle?
![Screenshot 2021-06-30 at 06 30 09](https://user-images.githubusercontent.com/49716569/123898145-77d45980-d96d-11eb-8bf8-167b87cbf90e.png)

**Comment by qinyihao** [comment]
@Eddievin You can try to install the insiders version of vscode, I installed copilot on the insiders version and it didn't have a popup message like you do.

**Reply by Eddievin** [reply]
Thank you.

**Comment by botatoes** [comment]
Sign up for the waitlist here https://github.com/features/copilot/signup
You will not be able to use GitHub Copilot until you are off of the waitlist.

**Reply by Eddievin** [reply]
Done. For how long do I have to be on the waiting list?
Thank you.

**Reply by arslannoorpansota** [reply]
I got mine invitation for preview within 36 hours!

**Reply by ghost** [reply]
How long it takes to get an invitation to GitHub copilot preview

**Reply by azartheen** [reply]
i signup copilot but it almost more than 5 days I am still on the waitlist!...  can i know the reason why i can't get access

**Reply by hassanhafiz44** [reply]
Using copilot today. Thanks.

**Reply by Eddievin** [reply]
How comes?   I have been on the waiting list for 2 weeks now no answer.  How did you receive the go ahead to use it?  Via Email??

**Reply by Eddievin** [reply]
> i signup copilot but it almost more than 5 days I am still on the waitlist!... can i know the reason why i can't get access

You're not alone Bro.

**Reply by hassanhafiz44** [reply]
I don't remember when I registered / signed up. Today, I received email saying you can download extension and I downloaded it. That's all.

**Comment by yahu1031** [comment]
![image](https://user-images.githubusercontent.com/35523357/124492476-234c3680-ddd2-11eb-86ce-58e05c31d678.png)
What is the issue? I have already signed up. What is the solution for this?

**Comment by qinyihao** [comment]
So why haven't I received the invitation yet?

Muhammad Hassan Javed ***@***.***> 于2021年7月14日周三 下午4:20写道：

**Reply by Eddievin** [reply]
Also been waiting for a while now. I dont know the criteria.

**Reply by hassanhafiz44** [reply]
> So why haven't I received the invitation yet? Muhammad Hassan Javed ***@***.***> 于2021年7月14日周三 下午4:20写道：
> […](#)
> Using copilot today. Thanks. — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#4415 (reply in thread)](https://github.com/github/feedback/discussions/4415#discussioncomment-1002687)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AJWDTMYYTDDQP7NZBUSL2XDTXVCD7ANCNFSM47ROTF2A> .

![Screenshot_20210714_160650_com google android gm](https://user-images.githubusercontent.com/24438937/125612307-4f796016-3473-4ccf-ab25-60b714c6d19c.jpg)

I don't know. I received the email today.

**Comment by michellemerrill** [comment, marked as answer]
As GitHub Copilot requires state-of-the-art AI hardware, we are offering it to a limited number of testers for free, during the technical preview. We want to use the preview to learn how people use GitHub Copilot and what it takes to operate it at scale. If the technical preview is successful, our plan is to build a commercial version of GitHub Copilot in the future.  To learn more about GitHub Copilot and our latest updates, please head over to https://copilot.github.com/.

**Comment by mobikats** [comment]
Just some quick feedback on Copilot, I've been using it for at least 10 hours of active programming at a guess, and I just wanted to share how great I am finding it - I keep catching myself muttering to myself "oh yes, so nice" whenever it fills in a bunch of boilerplate for me (in exactly the right way, requiring zero editing) :)

And I've had a few "WT..." moments too - for example I tend to write a comment describing my intent before writing the code (to help me think about it), and a couple of times CoPilot has then written the entire code section - matching the existing code for variable name patterns, style etc.

My love for this is growing daily!
